src/components/cardlist/CardList.js

```diff
@@ -0,0 +1,43 @@
+import SearchInput from "../searchbar/SearchInput";
+import "./CardList.css";
+import noImg from "../../assets/이미지 없을 때 배경.png";
+
+function CardList({ userData }) {
+  if (!userData) {
+    return <div>Loading...</div>;
+  }
+
+  const links = userData.links ?? "유저 링크";
+  return (
+    <div>
+      <SearchInput />
+      <div>
+        <ul>
+          {links.map(
```

Comment:
r: userData.links의 타입이 배열임을 가정한 코드로 보이는데 10번 줄에서 기본값으로 문자열을 할당하고 있네요 👀 문자열에서 map 메서드를 호출하면 에러가 발생할 것 같습니다. 기본값은 같은 배열 타입인 빈배열로 해주세요.

> Uncaught TypeError: "유저 링크".map is not a function

```suggestion
  const links = userData.links ?? [];
  
  return (
    <div>
      <SearchInput />
      <div>
        <ul>
          {links.map(
```